internal/workflow/preprocessing.go

```diff
@@ -53,17 +52,7 @@ func (w *PreprocessingWorkflow) Execute(ctx temporalsdk_workflow.Context, params
 	}()
 
 	localPath := filepath.Join(w.sharedPath, filepath.Clean(params.RelativePath))
-
-	// Extract package.
-	var extractPackageRes activities.ExtractPackageResult
-	e = temporalsdk_workflow.ExecuteActivity(withLocalActOpts(ctx), activities.ExtractPackageName, &activities.ExtractPackageParams{
-		Path: localPath,
-	}).Get(ctx, &extractPackageRes)
-	if e != nil {
-		return nil, e
-	}
-
-	removePaths = append(removePaths, localPath, extractPackageRes.Path)
+	removePaths = append(removePaths, localPath)
```

Comment:
I think we don't want to remove the localPath in this case, so we don't even need the remove paths activity in this repo ATM. I'm also having conflicts about removing it here or in Enduro's workflow even when the relative path changes, I think we'll figure that out when we implement the failed SIPs feature in Enduro, it may depend on where the extraction is happening (will follow-up in the email).

Reply:
Okay, I've removed the call to RemovePaths in the workflow for now, but kept the activity in anticipation of needing it in the future.